static/app/components/acl/featureDisabled.tsx

```diff
@@ -149,6 +149,10 @@ const HelpText = styled('p')`
   margin-bottom: ${space(1)};
 `;
 
+const StyledIconChevron = styled(IconChevron)`
+  margin-left: ${space(1)};
+`;
```

Comment:
thinking about layout primitives, I think it would be better if we’d style the parent component with `flex` and give that a `gap` between the children instead of giving a specific item a `margin-left`.

Reply:
This is slightly more complex unfortunately due to how buttons work, but also how chonk works. 

The reason why ButtonLabel was exported is because there is wrapper around the button children, however styled(button) only applies to the outer wrapper, leaving you with no way to target the children wrapping.

My solution to this was to use inherit rules from the parent [here](https://github.com/getsentry/sentry/pull/90243/files). I tried to apply styles and classname to the wrapper as well, but that ends up breaking the styles of pretty much every styled(button)